tests/www/views/test_views_connection.py

```diff
@@ -59,14 +59,25 @@ def test_create_connection(admin_client, session):
 
 
 def test_invalid_connection_id_trailing_blanks(admin_client, session):
```

Comment:
the connection id is not invalid anymore, we should rename the test:
```suggestion
def test_connection_id_trailing_blanks(admin_client, session):
```

Reply:
Good catch. Made changes